core/src/main/scala/scala/js/exp/TraitsExp.scala

```diff
@@ -35,7 +35,7 @@ trait TraitsExp extends Traits with ProxyExp {
     }
 
     val implClazz = Class.forName(traitClazz.getName + "$class")
-    val parents = traitClazz.getInterfaces.filter(_ != implicitly[Manifest[scala.ScalaObject]].erasure)
+    val parents = traitClazz.getInterfaces.filter(_ != implicitly[Manifest[AnyRef]].runtimeClass)
```

Comment:
Do [this test](https://github.com/js-scala/js-scala/blob/master/core/src/test/scala/scala/js/TestProxy.scala) still pass with your change?